docs/start/getting-started/fragments/ios/generate-model.md

```diff
@@ -1,22 +1,22 @@
 With the basic setup complete, next you will model the data your application will store. Amplify DataStore will use this model to persist data to your local device that will be synchronized to a backend API without writing any additional code. These models are specified as [GraphQL](http://graphql.org/) schemas. If you'd like, first [learn more](~/cli/graphql-transformer/overview.md) about GraphQL schemas and data modeling.
 
-1. In Xcode, make sure the project navigator is displayed. You can use **Cmd+1** to switch to this view. **Open the schema file** located at **Todo** > **AmplifyConfig** > **schema.graphql**.  In your project folder, this file is located at `"amplify/backend/api/amplifyDatasource/schema.graphql"`.
+1. In Xcode, make sure the project navigator is displayed. You can use **Cmd+1** to switch to this view. **Open the schema file** located at **Todo** > **AmplifyConfig** > **schema.graphql**. In your project folder, this file is located at `"amplify/backend/api/amplifyDatasource/schema.graphql"`.
```

Comment:
The position of the last period is weird.. not sure why it's showing up this way on my screen:
![Screen Shot 2020-09-23 at 12 52 41 PM](https://user-images.githubusercontent.com/1911939/94062587-bb28e880-fd9b-11ea-8d64-2c31a1f57a8d.png)

Are you seeing it as well?  If so, maybe we can add a newline to one of the sentences? :)

Reply:
weird, thanks for catching it. Let me see if I can fix it.